### ISSUE TITLE
possible misleading tip of learning rate?

### ISSUE THREAD

**Issue by zhuliyi0** [Linaqruf/kohya-trainer#188]
in the LoRA Dreambooth script LoRA and Optimizer Config section, it says "if you want to train with higher dim/alpha so badly, try using higher learning rate. Because the model learning faster in higher dim" but from experiment I observed that lower lr is needed for higher dim, which makes sense since higher dim is more prone to overfitting. Is there more that I am not aware of?

**Comment by Linaqruf** [comment]
Lol yeah, I mixed up on that part, I'll fix it later
